java/spacewalk-web.changes.bisht-richa.ux-ui-buttons-component

```diff
@@ -0,0 +1,2 @@
+- Deprecating the classes btn-success, btn-info, and btn-warning. 
```

Comment:
Here and below, I think we need different descriptions for the changelog. While this describes what's changed from the technical side, the changelog is for customers, not engineers. For a customer, things like `btn-success` etc don't say anything meaningful. I think something along the lines of "Updated user interfaces for pages X, Y, Z" or something similar would work.  

Maybe @deneb-alpha has a good suggestion, you're usually pretty good with changelog entries?